battle-engine.js

```diff
@@ -601,6 +601,20 @@ BattlePokemon = (() => {
 	};
 	BattlePokemon.prototype.transformInto = function (pokemon, user, effect) {
 		let template = pokemon.template;
+		if ((user && user.transformed && this.battle.gen >= 5) && (BattleSide.prototype.chooseMove)) {
```

Comment:
`(BattleSide.prototype.chooseMove)` seems trivially truthy and can be optimized out.